server/src/controllers/campaign.js

```diff
@@ -80,6 +80,63 @@ class CampaignController {
 
     res.status(200).send();
   };
+
+  static addStep = async (req, res) => {
+    const campaign = await db.Campaign.findOne({
+      where: {
+        id: req.params.id,
+        userId: req.currentUser.id
+      },
+      include: db.Step
+    });
```

Comment:
More efficient to do a Select COUNT SQL statement - do you know if Sequelize can do that?

Reply:
There is a Sequelize.fn but I couldn't get it to work with an association.